.ibm/pipelines/openshift-tests.sh

```diff
@@ -19,7 +19,7 @@ trap cleanup EXIT
 add_helm_repos() {
   helm version
 
-  declare -a repos=("bitnami=https://charts.bitnami.com/bitnami" "backstage=https://backstage.github.io/charts" "janus-idp=https://janus-idp.github.io/helm-backstage" "${HELM_REPO_NAME}=${HELM_REPO_URL}")
+  declare -a repos=("bitnami=https://charts.bitnami.com/bitnami" "backstage=https://backstage.github.io/charts" "rhdh-chart=https://redhat-developer.github.io/rhdh-chart" "${HELM_REPO_NAME}=${HELM_REPO_URL}")
```

Comment:
As we use 'rhdh-chart', we may omit to add redundant repository, "${HELM_REPO_NAME}=${HELM_REPO_URL}". Also, we may remove the environment property from the pipeline.

Reply:
note that the readme at that URL is still in progress... https://github.com/redhat-developer/rhdh-chart/pull/4